packages/grafana-ui/src/components/UnitPicker/UnitPicker.story.tsx

```diff
@@ -7,15 +7,25 @@ import { withCenteredStory } from '../../utils/storybook/withCenteredStory';
 
 import mdx from './UnitPicker.mdx';
 
+type Props = React.ComponentProps<typeof UnitPicker>;
```

Comment:
Instead of declaring `Props` separately here, it's better to import them from `UnitPicker.tsx`, this way we'll ensure the prop types are in sync.

Reply:
Thanks for the feedback @Clarity-89, it makes sense, landed in [3bf163f](https://github.com/grafana/grafana/pull/54694/commits/3bf163f3ea3bc0122176a0dde877aa904a7b6f6f)